Notebooks/Jumper.ipynb

```diff
@@ -335,24 +339,40 @@
    "source": [
     "rmseV_zos\n",
     "values = [rmseV_zos, rmseV_so.T, rmseV_thetao.T]\n",
-    "maps   = [rmseM_zos, rmseM_so, rmseM_thetao]\n",
-    "names  = [\"zos\", \"so\", \"thetao\"]\n",
+    "maps = [rmseM_zos, rmseM_so, rmseM_thetao]\n",
+    "names = [\"zos\", \"so\", \"thetao\"]\n",
     "colors = [\"tab:blue\", \"darkgreen\", \"darkred\"]\n",
     "\n",
-    "fig = plt.figure(figsize=(6,8))\n",
+    "fig = plt.figure(figsize=(6, 8))\n",
     "for i in range(3):\n",
     "    if i == 0:\n",
-    "        plt.errorbar(np.mean(values[i]), array.deptht[0], xerr=np.std(values[i]), fmt='.', label=names[i], color=colors[i], ecolor=\"grey\")\n",
+    "        plt.errorbar(\n",
+    "            np.mean(values[i]),\n",
```

Comment:
Is the space above this line ok?

Reply:
What space?

Reply:
My bad, I just saw the gap in the github window, there is not actual space (saw the line number now and yes there is no actual space)